package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "start": "yarn && react-native start",
     "run-android": "react-native run-android",
-    "run-ios": "react-native run-ios",
+    "run-ios": "cd ios && pod install && cd .. && react-native run-ios",
```

Comment:
Wouldn't it be slightly better to do the same as the **install:pod**?  
e.g. `cd ios && bundle install && bundle exec pod install && cd .. && react-native run-ios`

Reply:
Done, switched to using the other script